packages/radix/src/components/ToggleButton.tsx

```diff
@@ -52,35 +47,79 @@ export const ToggleButton = withTheme(
               zIndex: '5 !important', // TODO: Review in Primitives
               borderRadius: 1,
             },
           }),
-          { lineHeight: '1' },
+          variant({
+            size: {
+              0: {
+                height: 3,
+                padding: 0,
+                fontSize: 0,
+                minWidth: 4,
+                lineHeight: 1.1,
+                /**
+                 * TODO: Consider using box shadow for consistency
+                 *
+                 * This isn't too straightforward though. We can’t reliably center elements inside `ToggleButton` —
+                 * like same height icons — unless we remove the borders and use box-shadows instead, but also can’t
+                 * remove the borders and use box-shadows, because we use negative margins to compensate for adjacent
+                 * border.
+                 */
+                borderColor: 'transparent',
+                ':hover': {
+                  borderColor: 'transparent',
+                },
+                ':focus': {
+                  borderColor: 'blue500',
+                  boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
+                },
+              },
+              1: {
+                height: 5,
+                padding: 0,
+                fontSize: 1,
+                minWidth: 5,
+                lineHeight: 0,
+                borderColor: 'gray400', // TODO: Consider using box shadow for consistency
+                backgroundColor: 'gray100',
+                ':hover': {
+                  borderColor: 'gray500',
+                },
+                ':focus': {
+                  borderColor: 'blue500',
+                  boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
+                },
+              },
+            },
+          }),
+        ]}
+        activeCss={[
+          css({
+            fontWeight: 500,
+            letterSpacing: '-0.015em',
+            wordSpacing: '0.02em',
+            backgroundColor: 'blue100',
+            borderColor: 'transparent',
+            color: 'blue800',
+            borderRadius: 1,
+            boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
+            ':focus': {
+              borderColor: 'blue500',
+              boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
+            },
+          }),
         ]}
-        activeCss={css({
-          fontWeight: 500,
-          letterSpacing: '-0.025em',
-          wordSpacing: '0.02em',
-          backgroundColor: 'blue100',
-          borderColor: 'transparent',
-          color: 'blue800',
-          borderRadius: 1,
-          boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
-          '&:focus': {
-            boxShadow: `0 0 0 2px ${themeGet('colors.blue500')(props)}`,
-          },
-        })}
       />
     );
   })
 );
 
-ToggleButton.defaultProps = { variant: 'normal' };
+ToggleButton.defaultProps = {
+  size: 1,
```

Comment:
This is a bit odd,  for every component that supports size, the small version is always the default. Do you think this should be `0` for consistency sake?

Reply:
Nope. It's `1` so that default `<ToggleButton />` matches default `<Button />`, `<Input />`, and `<Select />`. It's primarily meant to be used at this size along other form elements.

At its heart, I rather see the inconsistency in how numbers for sizes mean nothing. Let's discuss when you are back.

Reply:
Ah, I see how you're thinking now 👍